plenum/server/client_authn.py

```diff
@@ -171,7 +171,7 @@ def authenticate(self,
 class CoreAuthMixin:
     # TODO: This should know a list of valid fields rather than excluding
     # hardcoded fields
-    excluded_from_signing = {f.SIG.nm, f.SIGS.nm}
+    excluded_from_signing = {f.SIG.nm, f.SIGS.nm, f.FEES.nm}
```

Comment:
A better way would be to register authenticators for each txn type and only include whitelisted fields for signing. But this makes developement tedious in general

Reply:
I believe the best way is to complete the Request/Message format re-factoring where signature is calculated in a deterministic way against payload, so there is no need to exclude anything.

But as a short-fix, why Plugins can't extend `excluded_from_signing ` if needed?

Reply:
Plugings should not be able to affect other plugins (at least we should aim for that). If a plugin can dictate what needs to be excluded from signing for all txns, then a plugin can (un)intentionally affect other plugins.

Reply:
What if we sign Fees as well? Does it break any workflows?

Reply:
Fees is intentionally excluded from signature since that free up the domain transaction author to care about payments, eg Alice might not have any sovrin tokens but still create a nym txn and send it to Bob who possesses tokens and he can attach the fees part to it and send it to ledger.

Reply:
But having plugins to add items to `exclude_from_signing` is not much worse than how it's done now (that the whole code is affected by something that just one plugin needs).
So, fees plugin can extend `exclude_from_signing`, and after Request format improvements, this will not be needed. I don't think that we will have many plugins before this refactoring that can be affected.